modules/acquia_cms_tour/composer.json

```diff
@@ -9,7 +9,7 @@
     },
     "require-dev": {
         "drupal/acquia_cms_place": "^1",
-        "drupal/google_analytics": "^4",
+        "drupal/google_analytics": "dev-3430780-automated-drupal-11",
```

Comment:
This module is obsolete I suggest we should to google_tag

Reply:
Yes, make sense, but will handle this in a separate ticket.